lib/rules/declaration-empty-line-before/__tests__/index.mjs

```diff
@@ -45,6 +45,37 @@ testRule({
 			message: messages.expected,
 			line: 1,
 			column: 5,
+			endLine: 1,
+			endColumn: 15,
+		},
+		{
+			code: 'a { top: 15px }',
+			fixed: 'a {\n\n top: 15px }',
```

Comment:
[ask] Doesn't this case duplicate with the case at L43 (`a { top: 15px; }`)?

Reply:
No it doesn't, there's a semicolon missing.
It matters for regression detection.
e.g. if someone eventually passes an `endIndex`
See 6cfca7d8.